core/constants.yaml

```diff
@@ -165,7 +165,7 @@ qanet_tns: &qanet_tns
 travis: &travis
   <<: *qanet_tns
 
-  k: 3  # it's needed for tests
+  k: 300  # it's needed for tests
```

Comment:
I decreased it to `35`, but it's temporary and I will set it back to `3` in a different task.